### ISSUE TITLE
Enable user to stop download file that's in progress

### ISSUE THREAD

**Issue by redshiftzero** [freedomofpress/securedrop-client#417]
This is not necessarily for beta, just a thought after reading @creviera's [comment here](https://github.com/freedomofpress/securedrop-client/issues/410#issuecomment-501065676): it would be good to enable the user to stop a long running file download, e.g. with a cross icon or something next to an in-progress download

**Comment by ninavizz** [comment]
Yep, that's been factored into the richer "Files Pane" design for files management. The "Pre Beta" design proposal for files will not allow for this, nor will it allow for single file deletion. 

If a user clicks the cancel-x icon on any of the in-progress files shown, the state of those slots goes back to not downloaded: https://zpl.io/a7We0Op

**Comment by eloquence** [comment]
From 2022-08-15 review with @gonzalo-bulnes @eloquence:
- Still a reasonable feature suggestion, would need further UX exploration, keeping open.
